openedx/features/subscriptions/api/v1/tests/test_models.py

```diff
@@ -0,0 +1,136 @@
+from datetime import date, datetime, timedelta
+import ddt
+import pytest
```

Comment:
@rehan99000 remove this `pytest` import too

Reply:
done